.prettierignore

```diff
@@ -0,0 +1,13 @@
+# Exclude everything by default
+/**/*
+
+# Include all folders
+!/**/*/
+
+# But exclude node_modules folders anywhere
+node_modules/
+
+# Include specific file extensions if they appear under a "src" folder
+!/src/**/*.ts
+!/src/**/*.tsx
+!/src/**/*.js
```

Comment:
When you include all folders, do you not get these included by default?

Reply:
it's a little counter intuitive for sure but no, you need _both_ statements (include all folders and don't exclude these files) to get the result we want 🤷

Reply:
I've added a comment to `.prettierignore` with a link to a Github discussion talking about how this file works